### ISSUE TITLE
windows 10 npm install fails

### ISSUE THREAD

**Issue by travisstaloch** [TulipCharts/tulipnode#36]
node v8.9.4
npm  v5.6.0

I am trying to use tulind on windows from zenbot.  I have been able to build and install talib after installing the 2017 c++ msbuild tools with visual studio installer and `npm install windows-build-tools`.  And that is working fine.  However, when I  try `npm install tulind` I get: 
> npm ERR! code UNKNOWN
> npm ERR! UNKNOWN: unknown error, open 'E:\Users\Travis\Documents\Code\zenbot-traviss\node_modules\.bin\rc'

I tried `npm install tulind --build-from-source` and that gives the same error. 
Also, I looked for windows precompiled binaries but couldn't find any links to those.  
Any ideas what might cause this?  If not, any advice for getting tulind running somehow? 
Thanks for any help.

**Comment by codeplea** [comment]
Can you link to a build log?

The precompiled binaries should download automatically if available.

**Comment by travisstaloch** [comment]
Thank you for the quick reply.  I was able to get tulind installed.  The problem was that some of the previously installed tulind files/folders were marked as readonly and could not be deleted or edited.  I must have used windows ubuntu subsystem to build them before.  That is the only thing I can think of that would have caused them to be readonly.  

I'm not sure how they became readonly.  First I tried clearing the npm cache, then eventually deleting node_modules altogether and there were tulind and forex.analytics files that couldn't be deleted.
